media-proxy/tests/mesh_context_tests.cc

```diff
@@ -0,0 +1,51 @@
+#include <gtest/gtest.h>
+#include "mesh/concurrency.h"
+
+using namespace mesh;
+
+class test_class {
+  public:
+    context::Context _ctx;
+
+    void init(context::Context &ctx) { _ctx = context::WithCancel(ctx); }
+};
+
+/**
+ * Test designed to detect memory leaks in the Context class.
+ *
+ * Run the test with Valgrind to detect memory leaks.
+ * valgrind --leak-check=full ./media_proxy_unit_tests --gtest_filter=*Context*
+ */
+TEST(MeshContext, constructor) {
+    auto ctx = context::WithCancel(mesh::context::Background());
```

Comment:
```suggestion
    auto ctx = context::WithCancel(context::Background());
```

Reply:
Fixed in https://github.com/OpenVisualCloud/Media-Communications-Mesh/pull/269/commits/c89d2973e0e8b88369250f85b1524cb66d0ae875